.github/workflows/ci.yml

```
@@ -0,0 +1,44 @@
+name: CI
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    strategy:
+      matrix:
+        platform: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: install
+        run: npm install
+      - name: build
+        run: npm run build
+      - name: test:win
+        if: matrix.platform == 'windows-latest'
+        run: npm run test:win
+      - name: test
+        if: matrix.platform != 'windows-latest'
+        run: npm run test:ci
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@master
```

I think you may need to build again here. Because jobs do not automatically persist the builds in Actions

```suggestion
      - uses: actions/checkout@master
      - name: Use Node.js 14.x
        uses: actions/setup-node@v1
        with:
          node-version: 14.x
      - name: install
        run: npm install
      - name: build
        run: npm run build
```

---

👌🏼